src/deploy/scope_managementgroup.ts

```diff
@@ -15,6 +15,7 @@ export async function DeployManagementGroupScope(azPath: string, location: strin
         templateLocation ?
             templateLocation.startsWith("http") ? `--template-uri ${templateLocation}`: `--template-file ${templateLocation}`
         : undefined,
+        managementGroupId ? `--management-group-id ${managementGroupId}` : undefined,
         deploymentMode ? `--mode ${deploymentMode}` : undefined,
         deploymentName ? `--name ${deploymentName}` : undefined,
         parameters ? `--parameters ${parameters}` : undefined
```

Comment:
is it possible to use parameter file uri just like template file uri?
I am just wondering if user would want to specify both files as uri.

Reply:
Should work just normally as the CLI accepts for parameters Uris.